dashboard/client/src/service/placementGroup.ts

```diff
@@ -0,0 +1,14 @@
+import { PlacementGroup } from "../type/placementGroup";
+import { get } from "./requestHandlers";
+
+export const getPlacementGroup = () => {
+  return get<{
+    result: boolean;
+    message: string;
+    data: {
+      [result: string]: {
+        [result: string]: Array<PlacementGroup>;
+      };
+    };
+  }>("api/v0/placement_groups?detail=1");
+};
```

Comment:
can we move these types into the placementGroups.d.ts file

Reply:
or if this a common for all state-api responses, maybe we can create a generic common type

`type StateApiResponse<PlacementGroups>`

Reply:
will do